.changeset/afraid-experts-eat.md

```diff
@@ -0,0 +1,51 @@
+---
+'@clerk/backend': major
+---
+
+Drop `user` / `organization` / `session` from auth object on **signed-out** state (current value was `null`). Eg
+
+```diff
+    // Backend
+    import { createClerkClient } from '@clerk/backend';
+
+    const clerkClient = createClerkClient({...});
+    const requestState = clerkClient.authenticateRequest(request, {...});
+
+    - const { user, organization, session } = requestState.toAuth();
+    + const { userId, organizationId, sessionId } = requestState.toAuth();
+
+    // Remix
+    import { getAuth } from '@clerk/remix/ssr.server';
+    
+    - const { user, organization, session } = await getAuth(args);
+    + const { userId, organizationId, sessionId } = await getAuth(args);
+
+    // or 
+    rootAuthLoader(
+        args,
+        ({ request }) => {
+            - const { user, organization, session } = request.auth;
+            + const { userId, organizationId, sessionId } = request.auth;
+            // ...
+        },
+        { loadUser: true },
+    );
+
+    // NextJS
+    import { getAuth } from '@clerk/nextjs/server';
+    
+    - const { user, organization, session } = getAuth(args);
+    + const { userId, organizationId, sessionId } = getAuth(req, opts);
```

Comment:
Just checking, did we have `getAuth.user` before v5? 
Nitpick, but this should probably be on its own changeset targeting nextjs (similar for remix and gatsby)

Reply:
Yes we had them in v4.
signed-in: https://github.com/clerk/javascript/blob/c59a2d4a22166076340b3c5c5e20c112a138acbe/packages/backend/src/tokens/authObjects.ts#L120
signed-out: https://github.com/clerk/javascript/blob/c59a2d4a22166076340b3c5c5e20c112a138acbe/packages/backend/src/tokens/authObjects.ts#L142

The change is applied only in the backend package and the changeset will add this change in the changelog of all the packages depending on backend, that's why i didn't add both remix, nextjs, gatsby.
@nikosdouvlis Do we want for each breaking change in backend to also add the other packages?